plugins/pastefromword/filter/default.js

```diff
@@ -1172,25 +1172,21 @@
 
 		removeSymbolText: function( element ) { // ...from a list element.
 			var symbol = element.attributes[ 'cke-symbol' ],
-				removed,
-				matched;
-
-			element.forEach( function( node ) {
-				// Since symbol may contains special characters we use `indexOf` (instead of RegExp) which is sufficient (#877).
-				// Stop after after first replaced string (#2690).
-				if ( !matched && node.value.indexOf( symbol ) > -1 ) {
-					matched = true;
-					node.value = node.value.replace( symbol, '' );
-
-					if ( node.parent.getHtml().match( /^(\s|&nbsp;)*$/ ) ) {
-						removed = node.parent !== element ? node.parent : null;
-					} else if ( !node.value ) {
-						removed = node;
-					}
+				node = element.findOne( function( node ) {
+					return node.value && node.value.indexOf( symbol ) > -1;
+				}, true ),
```

Comment:
Code style issue – there should be one more indentation level.

Reply:
Adding indentation level would make closing bracket higher indentation than opening, shouldn't they be the same?

Reply:
It's a very similar case to objects in such lists and they are indented with that extra tab: https://github.com/ckeditor/ckeditor-dev/blob/1f365357c043a74e69ad21f32c829929d8610106/plugins/tableselection/plugin.js#L893-L901